fili-core/src/test/groovy/com/yahoo/bard/webservice/data/config/metric/makers/ArithmeticMakerSpec.groovy

```diff
@@ -54,7 +53,7 @@ class ArithmeticMakerSpec extends Specification {
         ArithmeticMaker roundUpDivisionMaker = new ArithmeticMaker(
                 metricDictionary,
                 ArithmeticPostAggregation.ArithmeticPostAggregationFunction.DIVIDE,
-                (Function) { String it -> return new SketchRoundUpMapper() }
+                (Function) { String name -> return new SketchRoundUpMapper(name) }
```

Comment:
SketchRoundUpMapper has two single argument constructors.  It needs a type to choose one.